src/components/EventForm/index.tsx

```diff
@@ -66,6 +75,15 @@ export const EventForm: React.FC<EventFormProps> = ({ event }) => {
     }
   };
 
+  const handleisDeleting = () => {
```

Comment:
`handleIsDelecting` - capital i.

I would call this (and the state) openDeleteConfirmation and `isConfirmationModalOpen`